CHANGELOG.md

```diff
@@ -279,6 +279,7 @@
 - Ingest RS GCRF data from IATI
 - Ingest BA GCRF data from IATI
 - Ingest RAEng GCRF data from IATI
+- Reports cannot be unique for the Level A activity
```

Comment:
This might need rewriting with what the change *does* do...

Reply:
Yes, I found this wording hard to parse in the changelog and commit message. Maybe "Reports no longer have to be unique" rather than "Reports cannot be unique".